src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -146,10 +147,6 @@ const Stream = createReactClass({
     }
   },
 
-  shouldComponentUpdate(nextProps, nextState) {
```

Comment:
Why is this being removed and/or why was it there?

Reply:
It was added [here](https://github.com/getsentry/sentry/commit/02089216675478eb6a13dfeceb16d8e9aa737ef4)

I think removing this would help to simplify this component and we should avoid implementing custom `shouldComponentUpdate` functions unless there is a real performance benefit associated with it. I don't think this one meets this criteria since it just performs deep equality check on the entire state object anyway.